### ISSUE TITLE
react-native 0.43.3 compatibility

### ISSUE THREAD

**Issue by jjdp** [parse-community/Parse-SDK-JS#428]
in version 0.43.3, they renamed the `react-native.js` to` react-native-implementation.js` in `react-native/Libraries/react-native/`

i do not know how to fix this dynamically, without breaking to support < RN 0.43.3

**Comment by mxklsn** [comment]
I join, I encountered an error while updating:

`error: bundling: UnableToResolveError: Unable to resolve module `react-native/Libraries/react-native/react-native.js` from `react-native-project/node_modules/parse/lib/react-native/StorageController.react-native.js`: Module does not exist in the module map or in these directories: react-native-project/node_modules/react-native/Libraries/react-native`

 Could you tell us, do will support for this version? :(

**Comment by jjdp** [comment]
change the import in the meantime. package needs a version bump say to determine what RN to support

**Comment by mxklsn** [comment]
@jjdp 
I with my team did support Parse-SDK-JS for RN v0.43:
https://github.com/allmaxgit/Parse-SDK-JS

You can try this:

```
"dependencies": {
  ...
  "react": "16.0.0-alpha.6",
  "react-native": "^0.43.2",
  "parse": "git+https://github.com/allmaxgit/Parse-SDK-JS.git",
  ...
}

`watchman watch-del-all`
`rm -rf node_modules/ && npm i`
`npm start --reset-cache`

**Comment by thiagoghisi** [comment]
@mxklsn please create a PR with that change. I would love to see it in the next parse release.

**Comment by mmazzarolo** [comment]
@mxklsn thank you
@thiagoghisi yep, would be nice... I guess it would break the compatibility with the previous versions of React-Native though.

**Comment by alejonext** [comment]
Is the same https://github.com/parse-community/Parse-SDK-JS/issues/418

**Comment by adnbsr** [comment]
@mxklsn Thanks for your contribution. Could you also check [#435](https://github.com/parse-community/Parse-SDK-JS/issues/435) which Throws error just for `Android` ?

**Comment by peterpme** [comment]
:wave: anyway I can help push this along?

**Comment by felipemartim** [comment]
This seems to be fixed in 1.10.0.

**Comment by nesc58** [comment]
Yes. Fixed with 1.10.0.
